src/orders/components/OrderPriceLabel/OrderPriceLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import { Typography } from "@material-ui/core";
+import Money from "@saleor/components/Money";
+import { makeStyles } from "@saleor/macaw-ui";
+import { SearchOrderVariant_search_edges_node_variants_pricing } from "@saleor/orders/types/SearchOrderVariant";
+import * as React from "react";
+
+interface OrderPriceLabelProps {
+  pricing: SearchOrderVariant_search_edges_node_variants_pricing;
+}
+
+const useStyles = makeStyles(
+  theme => ({
+    percentDiscountLabelContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "flex-end",
+      justifyContent: "flex-end"
+    },
+    strike: {
+      textDecoration: "line-through",
+      color: theme.palette.grey[400],
+      fontSize: "smaller"
+    }
+  }),
+  { name: "OrderPriceLabel" }
+);
+
+const OrderPriceLabel: React.FC<OrderPriceLabelProps> = ({ pricing }) => {
```

Comment:
It would be good to find out if this is not a reusable component we might use outside of `orders` module, can you please check with @pwgryglak to see if that's the case.

Reply:
There is a similar implementation in `TableLine.tsx` but the styles are different and the discounted price text acts as a link there. I'll extract the two prices together to a separate component as it seems to be most likely reused somewhere else - I'll keep `OrderPriceLabel` because it's a wrapper and it will vary from the needed implementation.

Reply:
I see you extracted the code but no changes were made to `TableLine.tsx` file

Reply:
I used `TableLine.tsx` as an example that there was a single similar component but since it's a link I can't change it to use the new "reusable" component 😛 I extracted it for the future non-link "sale" prices